### PR TITLE
changed flakey tests for query-workbench

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description

added changes for workbench changes

### Issues Resolved

[List any issues this PR will resolve]

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by ruanyl** [comment]
@sumukhswamy The lint check failed, could you take a look? The query workbench release tests are failed as well.

**Comment by SuZhou-Joe** [comment]
@sumukhswamy Is the fix merged in main?

**Comment by sumukhswamy** [comment]
> @sumukhswamy Is the fix merged in main?

yes it has

**Comment by SuZhou-Joe** [comment]
I am going to merge this PR and wait to see if the related test cases pass in 2.x as main build has some error when launching.

**Review by SuZhou-Joe — APPROVED** [review]

File: cypress/integration/plugins/query-workbench-dashboards/ui.spec.js [hunk 1]
```diff
@@ -41,9 +39,9 @@ describe('Dump test data', () => {
 describe('Test PPL UI', () => {
   beforeEach(() => {
     cy.visit('app/opensearch-query-workbench');
-    cy.wait(QUERY_WORKBENCH_DELAY);
-    cy.get('.euiButton__text[title=PPL]').click({ force: true });
-    cy.wait(QUERY_WORKBENCH_DELAY);
+    cy.wait(1000);
```
Comment (SuZhou-Joe) [inline cypress/integration/plugins/query-workbench-dashboards/ui.spec.js:42]: Any reason why we replace the const `QUERY_WORKBENCH_DELAY` with 1000?

**Pull request closed without merging**